.github/workflows/deploy-frontend-pr-previews.yml

```diff
@@ -305,16 +305,16 @@ jobs:
       - name: Get preview service url
         id: preview-url
         run: |
-          read service_url < <(gcloud run services describe ${{ needs.branch-name.outputs.lowercase }}-next-platform-preview \
+          read service_url < <(gcloud run services describe ${{ needs.branch-name.outputs.lowercase }}-analytics-platform-preview \
             --format='value(status.url)' \
             --platform managed \
             --region ${{ secrets.REGION }})
           echo "url=${service_url}" >>$GITHUB_OUTPUT
 
-  next-platform-pr-comment:
-    name: next-platform-preview-link-comment
-    if: needs.check.outputs.run_next_platform == 'true'
-    needs: [next-platform]
+  analytics-platform-pr-comment:
+    name: analytics-platform-preview-link-comment
+    if: needs.check.outputs.run_analytics_platform == 'true'
+    needs: [analytics-platform]
     runs-on: ubuntu-latest
```

Comment:
[netmanager and platform product names](https://github.com/airqo-platform/AirQo-frontend/pull/960/commits/4160f33202ea22db08aea3344c56c199fd0b011b)